amr-wind/physics/ForestDrag.H

```diff
@@ -0,0 +1,90 @@
+#ifndef ForestDrag_H
+#define ForestDrag_H
+
+#include "amr-wind/core/Physics.H"
+#include "amr-wind/core/Field.H"
+#include "amr-wind/CFDSim.H"
+
+namespace amr_wind::forestdrag {
+
+namespace {} // namespace
+struct Forest
+{
+    amrex::Real m_type_forest;
+    amrex::Real m_x_forest;
+    amrex::Real m_y_forest;
+    amrex::Real m_height_forest;
+    amrex::Real m_diameter_forest;
+    amrex::Real m_cd_forest;
+    amrex::Real m_lai_forest;
+    amrex::Real m_laimax_forest;
+
+    amrex::Real calc_lm() const;
+};
+
+amrex::Real Forest::calc_lm() const
+{
+    amrex::Real treelaimax = 0.0;
+    if (m_type_forest == 2) {
+        amrex::Real ztree = 0;
+        amrex::Real expFun = 0;
+        amrex::Real ratio = 0;
+        amrex::Real treeZm = m_laimax_forest * m_height_forest;
+        const amrex::Real dz = m_height_forest / 100;
```

Comment:
is `/100` a typical thing?

Reply:
This is basically an integral and instead of doing any of the internal-based methods, I just used a simple sum rule with 100 points.

Reply:
ah. hmmm we should fix that. Lets deal with the other things first.